README.md

```diff
@@ -147,6 +147,79 @@ export class NotFoundComponent {
 }
 ```
 
+## Custom Webpack
+
+In some situations, it may be required to customize the `webpack` build while using `@nestjs/ng-universal`, especially when additional dependencies are include which include native NodeJS code.
```

Comment:
```suggestion
In some situations, it may be required to customize the `webpack` build while using `@nestjs/ng-universal`, especially when additional dependencies are included (that rely on native Node.js code).
```